### ISSUE TITLE
Invert U.S. citizen question in eligibility tool

### ISSUE THREAD

**Issue by trevorshannon** [theunitedeffort/theunitedeffort.org#229]
I think it makes more sense for the default (unchecked) state of the U.S. citizen question to mean "I am a U.S. citizen"

In other words, a user has to check a box if they are a citizen whereas we could invert the question such that the user only has to check a box if they are _not_ a citizen.

@rajivrajiv if this makes sense to you, do you think you could implement it?

**Comment by rajivrajiv** [comment]
Hi,

It makes more sense to check the box, if they are not a citizen. I will change it.

I had an appointment yesterday, and was away from the computer most of the day.

Rajiv Sidenur


From: ***@***.***>
Sent: Wednesday, January 11, 2023 22:53
To: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Subject: [philhawksworth/the-united-effort-orginization] Invert U.S. citizen question in eligibility tool (Issue #229)


I think it makes more sense for the default (unchecked) state of the U.S. citizen question to mean "I am a U.S. citizen"

In other words, a user has to check a box if they are a citizen whereas we could invert the question such that the user only has to check a box if they are not a citizen.

@rajivrajiv<https://github.com/rajivrajiv> if this makes sense to you, do you think you could implement it?

**Comment by trevorshannon** [comment]
Great, thank you!  Take your time, no problem.

**Comment by trevorshannon** [comment]
Fixed by #232
